<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::SplashScreen</class>
 <widget class="QDialog" name="tt3::gui::SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeTracker3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Objects/Tt3Large.png</normaloff>:/tt3-gui/Resources/Images/Objects/Tt3Large.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="tt3-gui.qrc">:/tt3-gui/Resources/Images/Objects/Tt3Large.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="actionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ACTION</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="tt3-gui.qrc">:/tt3-gui/Resources/Images/Objects/Tt3Large.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="contextLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CONTEXT</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="productLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TimeTracker3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="copyrightLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Copyright (C) 2026, Andrey Kapustin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="versionLabel">
     <property name="text">
      <string>Version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections/>
</ui>
